src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -844,10 +844,10 @@ def copy(
         >>> idx is new_idx
         False
         """
-        # TODO: SNOW-1458120 implement copy
-        WarningMessage.index_to_pandas_warning("copy")
         return Index(
```

Comment:
looks like we ignores the deep parameter when we do copy, please get a warning message for ignoring the parameters

Reply:
added